app/app/src/main/java/de/tu_darmstadt/seemoo/nexmon/gui/ToolsFragment.java

```diff
@@ -336,69 +290,33 @@ public void run() {
                         copyExtractedAsset(binInstallLocation, "tkiptun-ng");
                         copyExtractedAsset(binInstallLocation, "wesside-ng");
                         copyExtractedAsset(binInstallLocation, "wpaclean");
-
-                        tracker.send(new HitBuilders.EventBuilder()
-                                .setCategory("Tools")
-                                .setLabel("Aircrack")
-                                .setAction("install")
-                                .build());
                     }
 
                     if (chkNetcat.isChecked()) {
                         //toast("Installing netcat ...");
-                        copyExtractedAsset(binInstallLocation, "nc");
```

Comment:
Why did you remove this line?

Reply:
Oh, I went crazy on ctrl+k with nano, many analytics commands in that file.
Removing the last google file.
Got monitor mode active on my device by copying the firmware from bcmon application, no injection support is recognised but i can run the tools (at least on nougat, that's why I'm trying to get the minsdk to 21 or 22 to test on stock lollipop)